CppCoreGuidelines.md

```diff
@@ -20370,6 +20371,7 @@ Reference sections:
   Libraries used have to have been approved for mission critical applications.
   Any similarities to this set of guidelines are unsurprising because Bjarne Stroustrup was an author of JSF++.
   Recommended, but note its very specific focus.
+* _MISRA C++ 2008: Guidelines for the use of the C++ language in critical systems_. (Copies of this document can be found on the Internet, but it is not clear whether they are legally posted. This may limit its usefulness.)
```

Comment:
> Copies of this document can be found on the Internet

I assume by "the Internet", you mean https://www.misra.org.uk/Buyonline/tabid/58/Default.aspx

Reply:
Sorry if I wasn't clear on this point.

The link you point to is the paywall - although it is a very inexpensive document, the fact that there is any expense at all has a pretty strong negative effect. People just don't like to pay for stuff.

I was referring to copies like [this one](http://read.pudn.com/downloads669/sourcecode/embedded/2709598/MISRA-Cpp-2008-Unlocked.pdf) which are easy to find but not legal copies.

Reply:
I was just trying to hint at how inappropriate that sentence reads. 

> that raises the question as to whether it is appropriate to include a link to the commercially available copy

Where would we be if we couldn't reference  TC++PL and all other books listed in Bibliography?

Reply:
I don't really have a strong opinion on this, I am trying to defer to any existing practice for the document.

I never thought there was any problem with referencing commercial products, I was more questioning whether you want a link that takes you directly to a page asking you to purchase the document. 

I don't *think* any existing references are of that type, which is why I was uncertain about whether to include one here.

Really only hoping for guidance, not taking a stand.